build-support/bin/release.sh

```diff
@@ -32,6 +32,9 @@ readonly DEPLOY_PANTS_WHEEL_DIR="${DEPLOY_DIR}/${DEPLOY_PANTS_WHEELS_PATH}"
 # only pants core is included.
 : ${PANTS_PEX_PACKAGES:="pantsbuild.pants"}
 
+# URL from which pex release binaries can be downloaded.
+: ${PEX_DOWNLOAD_PREFIX:="https://github.com/pantsbuild/pex/releases/download"}
```

Comment:
Is this poking a hole for Twitter? It's probably harmless and I can't suggest a positive alternative solution but I will continue to point out its not awesome having to make private utility scripts support public apis. The pants script still has `WRAPPER_REQUIREMENTS` and `WRAPPER_SRCPATH` which may or may not still be used and at one point served similarly.

Reply:
Yeah, this is a hole for twitter; we want to be able to run `release.sh -w` in a locked down network, and sadly github releases are hosted on an unpredictable CDN :(